manifests/config.pp

```diff
@@ -38,6 +38,30 @@
     mode    => '0640',
   }
 
+  service { 'supervisord':
+    ensure => running,
+    enable => true,
+  }
+
+  package { 'supervisor':
+    ensure  => 'installed',
+    require => [
+      File['/etc/supervisord.d/laravel-worker-workflow.ini']
```

Comment:
It seems like the package should be installed first. what about before instead of require?

Reply:
fixed